Jellyfin.Server.Implementations/JellyfinDb.cs

```diff
@@ -62,30 +74,27 @@ public partial class JellyfinDb : DbContext
         public virtual DbSet<Series> Series { get; set; }
         public virtual DbSet<SeriesMetadata> SeriesMetadata { get; set; }
         public virtual DbSet<Track> Tracks { get; set; }
-        public virtual DbSet<TrackMetadata> TrackMetadata { get; set; }
-        public virtual DbSet<User> Users { get; set; } */
+        public virtual DbSet<TrackMetadata> TrackMetadata { get; set; }*/
 
-        /// <summary>
-        /// Gets or sets the default connection string.
-        /// </summary>
-        public static string ConnectionString { get; set; } = @"Data Source=jellyfin.db";
-
-        /// <inheritdoc />
-        public JellyfinDb(DbContextOptions<JellyfinDb> options) : base(options)
+        /// <inheritdoc/>
+        public override int SaveChanges()
         {
-        }
+            foreach (var saveEntity in ChangeTracker.Entries()
+                .Where(e => e.State == EntityState.Modified)
+                .OfType<ISavingChanges>())
```

Comment:
`Suspicious cast: there is no type in the solution which is inherited from both Microsoft.EntityFrameworkCore.ChangeTracking.EntityEntry' and 'Jellyfin.Data.ISavingChanges`

Additionally, the column `RowVersion` isn't being updated when I make changes to any of my users.

Reply:
Fixed with my latest commit